Documentation/network/servicemesh/mutual-authentication/installation.rst

```diff
@@ -0,0 +1,59 @@
+Prerequisites
+#############
+
+* Mutual authentication is only currently supported with SPIFFE APIs. 
+* By default, a SPIRE server is deployed when Cilium is installed.
+* Users have the option to disable the automatic deployment of the SPIRE server if they do not require mutual authentication.
+  Users can also disable the automatic deployment of the SPIRE server and can bring their own instead.
+  
+Installation
+############
+
+.. tabs::
+
+    .. group-tab:: Helm
+
+        Mutual authentication can be enabled with helm flag ``authentication.mutual.spire.enabled``
+        set as true. 
+        A SPIRE server can be deployed alongside it with the helm
+        flag ``authenticaton.mutual.spire.install.enabled`` also set as true.
+        Please refer to :ref:`k8s_install_helm` for a fresh installation.
+
+        .. parsed-literal::
+
+            $ helm upgrade cilium |CHART_RELEASE| \\
+                --namespace kube-system \\
+                --reuse-values \\
+                --set authentication.mutual.spire.enabled=true \\
+                --set authentication.mutual.spire.install.enabled=true
+
+            $ kubectl -n kube-system rollout restart deployment/cilium-operator
+            $ kubectl -n kube-system rollout restart ds/cilium
+
+        
+        Next you can check the status of the Cilium agent and operator:
+
+        .. code-block:: shell-session
+
+            $ cilium status
+
+        .. include:: ../../../installation/cli-download.rst
+
+    .. group-tab:: Cilium CLI
+
+        .. include:: ../../../installation/cli-download.rst
+
+        Mutual authentication and associated SPIRE server can be enabled with the below command.
+
+        .. code-block:: shell-session
+
+            $ cilium install \\
```

Comment:
@nvibert  Might be worth a note that this requires the Cilium CLI Helm mode enabled (`CILIUM_CLI_MODE=helm`). Otherwise, SPIRE will not be installed.